kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/MergeQueries.java

```diff
@@ -134,7 +140,25 @@ private void mergeFlush(
           batchNumber, intTable(intermediateTable));
       String mergeFlushQuery = mergeFlushQuery(intermediateTable, destinationTable, batchNumber);
       logger.trace(mergeFlushQuery);
-      bigQuery.query(QueryJobConfiguration.of(mergeFlushQuery));
+
+      int attempt = 0;
+      boolean success = false;
+      while (!success) {
+        try {
+          bigQuery.query(QueryJobConfiguration.of(mergeFlushQuery));
+          success = true;
+        } catch (BigQueryException e) {
+          if (BigQueryErrorResponses.isCouldNotSerializeAccessError(e)) {
+            attempt++;
+            if (attempt == 30) {
+              throw new BigQueryConnectException("After retrying running 30 times, it still failed", e);
```

Comment:
Lets add a bit of more information to the error message. 

`"Failed to merge rows to destination table within " + attempt + "  attempts"`